tests/Functional/starWarsData.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Digia\GraphQL\Test\Functional;
+
+function luke()
```

Comment:
Just a trivial thing, but I think you can name it like `Luke()`, `Vader()` because they are proper noun

Reply:
This was taken from the reference implementation as is.